docs/documentation.md

```diff
@@ -58,7 +58,7 @@ The library is open-source, based on [React.js](http://facebook.github.io/react/
 The fastest way to get started is to use a built version of **instantsearch.js** from a CDN:
 
 <div class="code-box">
-  <div class="code-sample-snippet js-toggle-snippet ignore">
+  <div class="code-sample-snippet ignore">

```

Comment:
I think there is a wrongly merged conflict here. The `js-toggle-snippet` class is needed for the correct switching of tabs.